packages/zoe/src/zoe.js

```diff
@@ -212,20 +227,26 @@ import { makeTables } from './state';
  * queries based on other information, we choose to omit it. For
  * instance, `installationHandle` can be derived from
  * `instanceHandle` and is omitted even though it is useful.
- * @param  {object} seat - an object defined by the smart
- * contract that is the use right associated with the invite. In
- * other words, buying the invite is buying the right to call
- * methods on this object.
- * @param  {object} customProperties - an object of
+ * @param  {OfferHook} offerHook - a function that will be handed the
+ * offerHandle at the right time, and returns a contract-specific
+ * OfferOutcome which will be put in the OfferResultRecord.
+ * @param  {CustomProperties} customProperties - an object of
  * information to include in the extent, as defined by the smart
  * contract
- * @returns {{ invite: Invite, inviteHandle: InviteHandle }}
+ * @returns {Invite}
+ *
+ * @callback OfferHook
+ * a function that will be handed the seat at
```

Comment:
Missed a seat.

A function that will be handed the `offerHandle`

(though "A function that will be handed" is itself strange phrasing)

Reply:
I'll replace it with

```js
/**
 * @callback OfferHook
 * This function will be called with the OfferHandle when the offer
 * is prepared. It should return a contract-specific "OfferOutcome"
 * value that will be put in the OfferResultRecord.
 * @param {OfferHandle} offerHandle
 * @returns {OfferOutcome}
 */
```

Reply:
added in 421d236c